### ISSUE TITLE
Add Preservica asset titles to child object caption fields upon ingest

### ISSUE THREAD

**Issue by ucancallmealicia** [yalelibrary/YUL-DC#2124]
**Story**

The Preservica XML metadata for each asset/TIF file contains a Title field, which includes a semantic filename for the asset, e.g.`ms_0397_s02_b006_f0083_0001`. This filename identifies the collection, series, box, and folder number of the item, as well as its position in the sequence of images. This naming convention is used for digitized paper and AV materials across YUL's Preservica instance.

It would be helpful to add the title field from the Preservica XML to the caption field upon ingest of Preservica images into DCS. Having this filename in the caption will facilitate adding structure to multi-foldered DCS objects post-ingest, as well as aid staff and patrons in identifying the location and sequence of images. Some units that are ingesting data into DCS are already adding similar data via Goobi: https://collections.library.yale.edu/catalog/32324421. 

The title should only be added to the caption upon the initial ingest from Preservica, not after a resync, as the caption field may have been changed in DCS and should not be overwritten.

Sample XML from Preservica API: https://yul-pres-tsdb.library.yale.edu/api/entity/information-objects/912fe52a-fa4c-407e-8883-4012c5d2811a

**Comment by martinlovell** [comment]
@ucancallmealicia 
It looks like InformationObjects and ContentObjects have titles.  I looked at a couple with Pattern 1, and the title of the IO matched the title of the CO.

Would it work to get the title from the ContentObject in all cases so it works with Pattern 1 and Pattern 2?

(Pattern 1: SO containing multiple IO's with representations, each with one CO.
 Pattern 2: Single IO containing representation with multiple CO.)

**Comment by ucancallmealicia** [comment]
Yes that works - the filename and title will always match, as far as I know. Thanks @martinlovell

**Comment by AKFosterDev** [comment]
PR ready for review - https://github.com/yalelibrary/yul-dc-management/pull/1026

**Comment by K8Sewell** [comment]
Deploying to test with release v2.59.44 - https://github.com/yalelibrary/yul-dc-management/releases/tag/v2.59.44

**Comment by K8Sewell** [comment]
Promoting to UAT for testing

**Comment by ucancallmealicia** [comment]
Looks good to me! Examples in UAT: https://collections-uat.library.yale.edu/management/parent_objects/900146415, https://collections-uat.library.yale.edu/management/parent_objects/900146368, https://collections-uat.library.yale.edu/management/parent_objects/900146338
